jans-lock/cedarling/tests/fetch-open-id-config.rs

```diff
@@ -0,0 +1,8 @@
+use cedarling::open_id_config;
+use wasm_bindgen_test::*;
+
+#[wasm_bindgen_test]
+async fn test() {
+	let config = open_id_config("https://account.gluu.org/.well-known/openid-configuration").await;
+	console_log!("Fetched config: {:?}", &config);
```

Comment:
Tests should be done through assertions, console logging isn't testing anything. Also, doing real HTTP requests during tests will be very performance intensive. We want to do request testing via mocking the responses in some way.

Reply:
Oh yeah, I saw the output and moved on. I'll add some assertions